untrusted_app.te

```diff
@@ -0,0 +1 @@
+allow untrusted_app camera_prop:file r_file_perms;
```

Comment:
You don't want to add any new untrusted_app rules.

Reply:
I tried to add this as system_app but google camera is always recognized as untrusted_app any idea ?

Reply:
Is this important?

Reply:
following this @luk1337 https://android.googlesource.com/device/google/wahoo/+/9da8401acba47d463d8f122525be7d26f686901e%5E%21/#F0

Reply:
So, it looks like Google now allows every domain to access camera_prop. See: `sepolicy/vendor/app.te`

Reply:
if you use google camera as stock cam app xD

Reply:
That's fine IMO, especially since they are using proper macro: r_file_perms vs get_prop.

Reply:
it always was recognized as untrusted_app anyway

Reply:
Uh, it doesn't matter if you do what Google did.

Reply:
any ideas how to manage this ?

Reply:
Do what Google did. Allow get_prop for appdomain.